lint/action.yaml

```diff
@@ -30,6 +30,10 @@ inputs:
   npm-token:
     required: false
     description: The Node Package Manager (npm) authentication token. This token is used to authenticate against the NPM registry.
+  internal-dep-prefix:
+    required: false
+    description: Internal Dependency Prefix. This is used to match internal dependencies and disallow beta versions.
```

Comment:
Specify here that it is a comma separated list of prefixes and maybe give an example